src/lib.rs

```diff
@@ -138,11 +139,15 @@ impl PrivateKeyRotationCheckResult {
         }
     }
 
-    //    pub fn group_rotation_needed() -> Option<Vec<GroupId>> {
-    //        unimplemented!()
-    //    }
+    pub fn group_rotation_needed(&self) -> Vec<GroupId> {
+        match &self.rotations_needed {
+            EitherOrBoth::Right(groups) | EitherOrBoth::Both(_, groups) => groups.to_vec(),
+            _ => vec![],
+        }
+    }
 
-    //    pub fn rotate_all(ironoxide: &IronOxide) -> (UserId, Vec<GroupId>) //TODO consider when group private key rotation is added
+    // TODO (reviewers): Is this something we want and I should implement as part of this ticket?
+    // pub fn rotate_all(ironoxide: &IronOxide) -> (UserId, Vec<GroupId>) //TODO consider when group private key rotation is added
```

Comment:
Unless this is something that NCLH requires, I would say we punt on implementing this.